src/main/java/org/alliancegenome/mati/service/SubdomainService.java

```diff
@@ -0,0 +1,20 @@
+package org.alliancegenome.mati.service;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.alliancegenome.mati.entity.SubdomainEntity;
+import org.alliancegenome.mati.repository.SubdomainRepository;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.List;
+
+@ApplicationScoped
+@AllArgsConstructor
+@Slf4j
```

Comment:
Quarkus has its own logging framework which might be easier than trying to setup @Slf4j you can use `Log.info("")` and it comes from the `import io.quarkus.logging.Log` package.

Reply:
Removed slf4j